sources/engine/Stride.Rendering/Rendering/Images/Fog/Fog.cs

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) Xenko contributors (https://xenko.com) and Silicon Studio Corp. (https://www.siliconstudio.co.jp)
+// Distributed under the MIT license. See the LICENSE.md file in the project root for more information.
+
+using System;
+using Stride.Core;
+using Stride.Core.Mathematics;
+using Stride.Graphics;
+
+namespace Stride.Rendering.Images {
+    /// <summary>
+    /// A fog filter.
+    /// </summary>
+    [DataContract("Fog")]
+    public class Fog : ImageEffect {
```

Comment:
Could you please fix the curly bracket so that they are on their own lines to follow rest of codebase?
Except that, LGTM, thanks for the contribution!